ext/opentelemetry-ext-opencensusexporter/src/opentelemetry/ext/opencensusexporter/metrics_exporter/__init__.py

```diff
@@ -121,13 +129,23 @@ def translate_to_collector(
             label_keys=label_keys,
         )
 
+        # If cumulative and stateful, explicitly set the start_timestamp to
+        # exporter start time.
+        if metric_record.instrument.meter.batcher.stateful:
+            start_timestamp = exporter_start_timestamp
+        else:
+            start_timestamp = None
+
         timeseries = metrics_pb2.TimeSeries(
             label_values=label_values,
             points=[get_collector_point(metric_record)],
+            start_timestamp=start_timestamp,
```

Comment:
What is used instead if `start_timestamp` is `None`?

Reply:
According to the proto docstring, but idk if this is true -- I would be happier to send it explicitly.

Is there any way to get the `start_timestamp` other than saving the previous end time from the last run?

Reply:
If this is done in the backend I think it is fine?